### PR TITLE
Add helm chart configuration to bundle

### PR DESCRIPTION
**Pull request by TerryHowe**
Add configurations to bundle versions so for each helm chart, we can have the associated valid configurations

### CODE REVIEW

File: api/v1alpha1/packagebundle_types.go [hunk 1]
```diff
@@ -83,6 +83,9 @@ type SourceVersion struct {
 
 	// Images is a list of images used by this version of the package
 	Images []VersionImages `json:"images,omitempty"`
+
+	// Configurations is a list of configurations used by this version of the package
```
Comment (ewollesen) [inline api/v1alpha1/packagebundle_types.go:87]: I wouldn't mind an added bit of commentary for how these are used.

**Review by ewollesen — APPROVED** [review]

File: api/v1alpha1/packagebundle_types.go (hunk 1 above)
Comment (TerryHowe) [reply api/v1alpha1/packagebundle_types.go]: Done

**Review by ewollesen — APPROVED** [review]
/lgtm

**Comment by TerryHowe** [comment]
> This is for overriding values in the helm charts? Or is this for something else.

These are the config values for the our package custom resources. Should map to a subset of the `values.yaml`

I wonder why this hasn't merged.

**Pull request merged**